### PR TITLE
Disabled pagedForgingStakes API after Forger Stakes V2 activation

### PR DESCRIPTION
**Pull request by saratnt**
## Description
Disabled pagedForgingStakes http API after Forger stakes V2 activaation.

## Jira Ticket
[EON-1852](https://horizenlabs.atlassian.net/browse/EON-1852?atlOrigin=eyJpIjoiMDU3NzVmNjY1YjExNDhmNjk4NjEzZmVkODkzZmYwYTYiLCJwIjoiaiJ9)

[EON-1852]: https://horizenlabs.atlassian.net/browse/EON-1852?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: sdk/src/main/resources/account/api/accountApi.yaml [hunk 1]
```diff
@@ -1500,7 +1500,7 @@ paths:
       tags:
         - transaction
       summary:  Returns the paginated list of forging stakes.
-      description: Returns the paginated list of forging stakes.
+      description: Returns the paginated list of forging stakes. It will be disabled after Forger Stake V2 will be activated.
```
Comment (i-skrypnyk) [inline sdk/src/main/resources/account/api/accountApi.yaml:1503]: Maybe change comment to something like "This endpoint is disabled after Forger Stake V2 activation.", so that it is not related to some point in time

**Review by i-skrypnyk — APPROVED** [review]

File: sdk/src/main/resources/account/api/accountApi.yaml (hunk 1 above)
Comment (saratnt) [reply sdk/src/main/resources/account/api/accountApi.yaml]: Done

**Review by paolocappelletti — APPROVED** [review]
